### ISSUE TITLE
InfluxDB integration in docker-compose

### ISSUE THREAD

**Issue by smstep** [brocaar/loraserver-docker#15]
Hi! I used your docker-compose.yml file to create a loraserver which works well.

I added two docker images to the docker-compose file, Grafana and InfluxDB. I tried to use the InfluxDB integration from the Loraserver webinterface.

My docker-compose.yml file, looks like this: 

```
version: "3"

services:
  loraserver:
    image: loraserver/loraserver:2
    volumes:
      - ./configuration/loraserver:/etc/loraserver

  appserver:
    image: loraserver/lora-app-server:2
    ports:
      - 8080:8080
    volumes:
      - ./configuration/lora-app-server:/etc/lora-app-server

  gatewaybridge:
    image: loraserver/lora-gateway-bridge:2
    ports:
      - 1700:1700/udp
    volumes:
      - ./configuration/lora-gateway-bridge:/etc/lora-gateway-bridge
  
  geoserver:
    image: loraserver/lora-geo-server:2
    volumes:
      - ./configuration/lora-geo-server:/etc/lora-geo-server
  
  postgresql:
    image: postgres:9.6-alpine
    volumes:
      - ./configuration/postgresql/initdb:/docker-entrypoint-initdb.d
      - postgresqldata:/var/lib/postgresql/data

  redis:
    image: redis:4-alpine
    volumes:
      - redisdata:/data

  mosquitto:
    image: eclipse-mosquitto
    ports:
      - 1883:1883

  influxdb:
    image: influxdb
    restart: always
    environment:
      - INFLUXDB_DB="database_test"
      - INFLUXDB_ADMIN_ENABLED="true"
      - INFLUXDB_ADMIN_USER="admin"
      - INFLUXDB_ADMIN_PASSWORD="admin"
      - INFLUXDB_USER="user"
      - INFLUXDB_USER_PASSWORD="user"
    ports:
      - 8086:8086
    volumes:
      - influxdb:/var/lib/influxdb

  grafana:
    image: grafana/grafana
    ports:
      - 3000:3000
    volumes:
      - grafana:/var/lib/grafana/data



volumes:
  postgresqldata:
  redisdata:
  influxdb:
  grafana:
```
I set up some configuration for the InfluxDB image. A new user is added and a database.
When  I log into Grafana at localhost:3000 I can set up a connection to my database_test which was created during running the docker-compose up command. In Grafana iÍ get immediately feedback if the connection to http//:influxdb:8086 (I am using dockers address space here)is successful.
I also get the following message from the InfluxDB log which confirms the connection:

```
ts=2019-04-15T12:31:40.459969Z lvl=info msg="Executing query" log_id=0EpFAvvG000 service=query query="SHOW RETENTION POLICIES ON database_test"
[httpd] 172.19.0.1, 172.19.0.1,172.19.0.5 - user [15/Apr/2019:12:31:40 +0000] "GET /query?db=database_test&epoch=ms&q=SHOW+RETENTION+POLICIES+on+%22database_test%22 HTTP/1.1" 200 149 "-" "Grafana/6.1.3" 6b6ec1cb-5f7a-11e9-8001-0242ac130004 4230
``` 

After I configured the Loraserver with gateway and lora nodes I received data but Loraserver couldn't connect to the InfluxDB database_test and write the data

I got following log entry from the lora appserver:
``` 
time="2019-04-15T13:04:39Z" level=error msg="integration/multi: integration *influxdb.Integration error" error="sending measurements error: expected 2xx response, got: 404 (404 page not found\n)"

``` 

What could be the reason why the Loraserver can't connect to the InfluxDB container?

I tested similar setup in ubuntu successfully before.

**Comment by smstep** [comment]
I used the mqtt integration and get the data by subscribing to the application topic. I wrote my own python script to write the data into the influxdb database. The script is running in a separate docker container. This works for me and is probably a better solution because i can subscribe also to the communication data of the gateway and store it in a different database. The actual infludb integration problem i stated earlier remains the same.
